### ISSUE TITLE
Three-class training vs binary validation data

### ISSUE THREAD

**Issue by hossein-amirkhani** [TalSchuster/FeverSymmetric#2]
I am using the version of FEVER dataset that you have processed and released at the following addresses:
https://www.dropbox.com/s/v1a0depfg7jp90f/fever.train.jsonl 
https://www.dropbox.com/s/bdwf46sa2gcuf6j/fever.dev.jsonl

The problem is that even though the training data is a three-class dataset, the validation data only includes "Support" and "Refute" samples. Can you please let me know what is the reason? Also, can we use this data as it is (training on three-class data and testing on binary data)?

**Comment by TalSchuster** [comment]
Hi @h-amirkhani, thanks for reaching out.

In this work, we focused on evaluating the sensitivity of fact verification models to changes flip the relation of a claim with an evidence sentence. For example from supporting to refuting or vise versa. That's why we focus the evaluation on these two classes. Another reason is that the FEVER dataset doesn't provide annotations for NEI evidence (as in theory almost every sentence that is not SUP or REF is NEI with respect to a claim), so in order not to condition our evaluation on the way one collects the NEI evidence, we simply not include them in the evaluation as they are anyway not what we're focusing on here.

In general, though, you might want your model to use the NEI relation when suitable. So we do train a three-way classifier. If you want to evaluate the overall performance, you can add NEI examples using any method you find suitable for retrieving the neutral evidence.

For that, you might also find our recent work useful: [https://github.com/TalSchuster/VitaminC](https://github.com/TalSchuster/VitaminC).
We created there a large dataset with contrastive evidence, including NEI. You can also find in that repository a processed version of FEVER with NEI for both train and dev, where we used BM25 to retrieve relevant articles and randomly selected a sentence from them as neutral evidence.

**Comment by hossein-amirkhani** [comment]
Thank you Tal for your thorough answer and for informing me about your recent work. VitaminC fact verification validation data includes 9042 NEI samples. Do you consider the following work sensible: To exploit the previous process training and validation data (https://www.dropbox.com/s/v1a0depfg7jp90f/fever.train.jsonl and
https://www.dropbox.com/s/bdwf46sa2gcuf6j/fever.dev.jsonl) augmenting the validation data with the 9042 NEI samples from VitaminC validation data?
In other words, the question is whether VitaminC validation data comes from the same distribution as the above data?

**Comment by TalSchuster** [comment]
The format of VitaminC is similar to FEVER but the collection and annotation process was different, as described in [our paper](https://arxiv.org/abs/2103.08541). So the distribution is not the same. But you can see in our results that training with it helps to improve the model's robustness.

In that repo, we also have a link to a version of FEVER with NEI evidence sentences that we retrieved:
https://github.com/TalSchuster/talschuster.github.io/raw/master/static/vitaminc_baselines/fever.zip

**Comment by hossein-amirkhani** [comment]
Great! Does this version of FEVER come from the same distribution as the "dropbox" version? Does it have any sample in common with the "dropbox" training data? I want to use your processed validation set (dropbox version) augmenting it with new NEI samples, along with the "dropbox" training data to train a model. 
I didn't find specific information regarding this version of FEVER in either the paper or the Github repo.

**Comment by TalSchuster** [comment]
The SUP and REF cases should overlap between the two. If what you are looking for is FEVER with NEI examples, maybe just using the fever.zip file above (from the VitaminC repo) should work?

**Comment by hossein-amirkhani** [comment]
Thanks

**Comment by yyhappier** [comment]
Hello, could you please share the processed training and evaluation data with me? The addresses are now invalid. Thank you very much!
